src/OSPSuite.Infrastructure.Import/Services/Importer.cs

```diff
@@ -281,7 +281,9 @@ DataImporterSettings dataImporterSettings
          dataSource.SetNamingConvention(configuration.NamingConventions);
          var sheets = new Cache<string, DataSheet>();
          var missingSheets = new List<string>();
-         foreach (var key in configuration.LoadedSheets)
+         var sheetList = dataImporterSettings.IgnoreSheetNamesAtImport ? dataSourceFile.DataSheets.Keys : configuration.LoadedSheets;
+
```

Comment:
what will happen if `IgnoreSheetNamesAtImport = true ` and the file contains e.g. empty sheets (or sheets which do not match the configuration)?

Reply:
In such a case, I expect exceptions on line 297 (AddSheets) but this is how it should be right? we let the exception raise and catch on the R side. We still need to discuss how to catch